content/backend/graphql-scala/4-deferred_resolvers.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Deferred Resolvers
+pageTitle: "Usage of Deferred Resolvers for better performance"
+description: "In this chapter, at first you will learn how to use deferred resolvers for better performance."
```

Comment:
The other way round? Put the benefit first then the way to get it. Something like `In this chapter you will learn how to improve performance with deferred resolvers`